packages/fetchai/connections/p2p_libp2p/go.sum

```diff
@@ -20,6 +20,7 @@ github.com/btcsuite/goleveldb v0.0.0-20160330041536-7834afc9e8cd/go.mod h1:F+uVa
 github.com/btcsuite/snappy-go v0.0.0-20151229074030-0bdef8d06723/go.mod h1:8woku9dyThutzjeg+3xrA5iCpBRH8XEEg3lh6TiUghc=
```

Comment:
@lrahmani please check if there is a way to automatically check for licenses in the dependencies. We do this on the python side and it is quite important obviously.

Reply:
good remark, will look into it